KDEpy/utils.py

```diff
@@ -131,8 +131,75 @@ def autogrid(data, boundary_abs=3, num_points=None, boundary_rel=0.05):
     return cartesian(list_of_grids)
 
 
+def weighted_std(values, weights, ddof=0):
+    """Return the weighted standard deviation.
+
+    Examples
+    --------
+    >>> weighted_std([1, 2, 2], weights=[1, 1, 1])
+    0.4714045207910317
+    >>> weighted_std([1, 2], weights=[1, 2])
+    0.4714045207910317
+    >>> weighted_std([1, 2, 2], weights=[1, 1, 1], ddof=1)
+    0.5773502691896257
+    >>> weighted_std([1, 2], weights=[1, 2], ddof=1)
+    0.5773502691896257
+
+    """
+    values = np.asarray(values)
+    weights = np.asarray(weights)
+    assert np.all(weights > 0), "All weights must be >= 0"
```

Comment:
Nit: the error message doesn't match the assertion - one is `>` and the other is `>=`